src/grader/mutation_test.mli

```diff
@@ -0,0 +1,110 @@
+(* This file is part of Learn-OCaml.
+ *
+ * Copyright (C) 2019 OCaml Software Foundation.
+ * Copyright (C) 2016-2018 OCamlPro.
+ *
+ * Learn-OCaml is distributed under the terms of the MIT license. See the
+ * included LICENSE file for details. *)
+
+type 'a test_result =
```

Comment:
This looks like a quite general type definition. Shouldn't it be in another module?

Reply:
Hmm, it is probably not necessary to expose this type and the associated `run_test_against` functions from this module at all, since it is basically just a weaker version of the test functions in `Test_lib`.